enterprise/tasks.py

```diff
@@ -255,16 +255,18 @@ def send_group_membership_invitation_notification(
                 'user_id': group_membership.enterprise_customer_user.user_id
             })
     recipients = []
-    for pecu_email in pecu_emails:
-        recipients.append(braze_client_instance.create_recipient_no_external_id(pecu_email))
-    braze_client_instance.create_braze_alias(
-        [pecu_emails],
-        ENTERPRISE_BRAZE_ALIAS_LABEL,
-    )
-    for ecu in ecus:
-        recipients.append(braze_client_instance.create_recipient(
-            user_email=ecu['user_email'],
-            lms_user_id=ecu['user_id']))
+    if len(pecu_emails) > 0:
+        for pecu_email in pecu_emails:
+            recipients.append(braze_client_instance.create_recipient_no_external_id(pecu_email))
+        braze_client_instance.create_braze_alias(
+            pecu_emails,
+            ENTERPRISE_BRAZE_ALIAS_LABEL,
+        )
+    if len(ecus) > 0:
```

Comment:
I don't think this check is necessary- if `ecus` is empty, it will just not iterate through the loop